cpu/nrf52/include/periph_cpu.h

```diff
@@ -180,6 +186,34 @@ typedef struct {
 } uart_conf_t;
 #endif
 
+/**
+ * @brief  SPI configuration values
+ */
+typedef struct {
+    NRF_SPIM_Type *dev; /**< SPI device used */
+    gpio_t sclk;        /**< CLK pin */
+    gpio_t mosi;        /**< MOSI pin */
+    gpio_t miso;        /**< MISO pin */
+    uint8_t ppi;        /**< PPI channel */
```

Comment:
This is only required for the errata58 workaround and thus not required for the nrf52840 MCU. Should I ifdef this to only use it with the nrf52832?

Reply:
It's also not required for the nRF52811

Reply:
I'd say so - but I'd suggest to ifdef for the errata, not the CPU family so when adding new members of the family there only has to be a changed in one place.

Reply:
Done